src/AccessibilityInsights/MainWindowHelpers/AutoUpdate.cs

```diff
@@ -122,9 +123,15 @@ private UpdateDialog CreateUpgradeDialog(Uri releaseNotesUri, AutoUpdateOption u
         private async void DownLoadInstaller(IAutoUpdate autoUpdate, AutoUpdateOption updateOption)
         {
             UpdateResult result = UpdateResult.Unknown;
+
+            // The UAC prompt from the version switcher will appear behind the main window
+            // if it is topmost, we store, change, and restore the value in this method.
```

Comment:
It's not clear what "it" refers to hear. Is it the UAC dialog or AI_Win.

Reply:
I had been advised that it refers to the preceding noun phrase, which is the main window in this case. Upon further research, I can find no evidence that this is a rule. Updating comment and my brain accordingly.